library/edi.inc

```diff
@@ -323,7 +325,7 @@ function print_elig($res, $X12info, $segTer, $compEleSep)
             $PATEDI  .= create_NM1($row, 'PR', $X12info, $segTer, $compEleSep);
             // For Provider Segment
             $PATEDI  .= create_HL($row, 2, $X12info, $segTer, $compEleSep);
-            $PATEDI  .= create_NM1($row, 'FA', $X12info, $segTer, $compEleSep);
+            $PATEDI  .= create_NM1($row, 'PI', $X12info, $segTer, $compEleSep);  // 5010 no longer uses FA
```

Comment:
I could have swore I already commented on this! Anyway, this can't be PI but either 1P, FA or 80. I used 1P in the realtime 270.

Reply:
oh, good catch, PI 1P, dyslexia is kinda contagious :)